postgresql.conf.jinja

```diff
@@ -300,6 +300,12 @@ log_min_duration_statement = '1500ms'
 log_line_prefix = '%m [%p] %q%u@%d '
 log_timezone = 'Europe/Berlin'
 
+# Specfies queries to be logged, log destination and log level
```

Comment:
In most other statements, I copy the respective section from the postgresql configuration file to explain the setting.
Feel free to add the sections as well, unless you think they are super self-explanatory. At least for `logging_collector` I'd like the [documentation](https://www.postgresql.org/docs/11/runtime-config-logging.html#GUC-LOGGING-COLLECTOR) because it can have some sideeffects.